app/src/interfaces/input-rich-text-html/index.ts

```diff
@@ -284,5 +284,18 @@ export default defineInterface({
 				interface: 'input',
 			},
 		},
+		{
+			field: 'publicUrl',
+			name: 'public url',
+			type: 'boolean',
+			meta: {
+				note: 'Use public url for images in this wysiwyg container',
```

Comment:
We should use the `$t:` syntax here and put the translations in the en-US.yaml file so it's a translated value

Reply:
updated to support translations.